tests/unit/test_step.py

```diff
@@ -732,10 +732,10 @@ def test__duplicate_template_step(
     mocker.patch("easylink.implementation.Implementation.validate", return_value=[])
     if step_type == "loop":
         step = LoopStep(**loop_step_params)
-        config_key = "iterate"
+        config_key = "iterations"
     else:  # cloneable
         step = CloneableStep(**cloneable_step_params)
-        config_key = "parallel"
+        config_key = "clones"
```

Comment:
whoops, missed this when I changed 'parallel' to 'clones'. This isn't a great test